server/services/config/crm14DisplayConfig.json

```diff
@@ -338,7 +338,7 @@
       "title": "Evidence: 1",
       "subsections": [
         {
-          "title": "Questions to inform evidential requirements",
+          "title": "Questions to Inform Evidential Requirements",
```

Comment:
Did you check with `Prince` for Sentence case? -  `Inform evidential requirements`

Reply:
This was the case specified on the defect ticket. I will check with him again

Reply:
Prince has confirmed that the text should be - Questions to Inform Evidential Requirements